src/npcmove.cpp

```diff
@@ -3741,9 +3741,9 @@ void npc::pick_up_item()
             worst_item_value = itval;
         }
         i_add( it );
+        mod_moves( -get_speed() );
```

Comment:
Note: This is a functional change. They lose a turn for every item they pick up, as opposed to simply 1 turn regardless of how many they pick up.